### ISSUE TITLE
I can't import the richMenuAlias module

### ISSUE THREAD

**Issue by TF-Takishita** [line/line-bot-sdk-python#347]
## request
I need you to add a `RichMenuAlias` to the code below.

https://github.com/line/line-bot-sdk-python/blob/c0b7488c2182c527f41a7d4cff603d0b500cde82/linebot/models/__init__.py#L165-L170

## System Informations

* Python version: 3.8
* SDK version: 1.20.0 , 2.0.1

## What I want to do
`from linebot.models import RichMenuAlias`

### Logs
```
[ERROR] Runtime.ImportModuleError: Unable to import module 'app': cannot import name 'RichMenuAlias' from 'linebot.models' (/var/task/linebot/models/__init__.py)
Traceback (most recent call last):
```

**Comment by TF-Takishita** [comment]
@louis70109 
I requested it in Pull Request #355.
Thanks for the reply. :)

**Comment by TF-Takishita** [comment]
@cryeo 
I'll give it a try.
Thank you. :)

**Comment by Mraldossri** [comment]
#355 from .rich_menu import (  # noqa
    RichMenu,
    RichMenuSize,
    RichMenuArea,
    RichMenuBounds,
)

**Comment by louis70109** [comment]
Closed cause #355
